cloudos_cli/__main__.py

```diff
@@ -3606,5 +3610,192 @@ def list_images(ctx,
         click.echo(f"[ERROR] {str(e)}", err=True)
 
 
+@images.command(name="set")
+@click.option('-k',
+              '--apikey',
+              help='Your CloudOS API key.',
+              required=True)
+@click.option('-c',
+              '--cloudos-url',
+              help=(f'The CloudOS url you are trying to access to. Default={CLOUDOS_URL}.'),
+              default=CLOUDOS_URL,
+              required=True)
+@click.option('--procurement-id', help='The specific CloudOS procurement id.', required=True)
+@click.option('--organisation-id', help='The Organisation Id where the change is going to be applied.', required=True)
+@click.option('--image-type', help='The CloudOS resource image type.', required=True)
+@click.option('--provider', help='The cloud provider. Only aws is supported.', required=True)
+@click.option('--region', help='The cloud region. Only aws regions are supported.', required=True)
+@click.option('--image-id', help='The new image id value. Required.', required=True)
+@click.option('--image-name', help='The new image name value. Optional.', required=False)
```

Comment:
`click` makes clear the required and the optional params. We don't need to add it to the help string